CODEOWNERS

```diff
@@ -0,0 +1 @@
+/packages/schema/ @Mearman
```

Comment:
Not sure we want this. It means that you have to approve any change we make to schema. I value your opinion very much, but I'm not sure it should be blocking :)

Reply:
Oh I thought on GitHub by default it just pinged the owner on touching PRs will read the docs more

Reply:
Yeah so it's an optional addition to require codeowner signoff

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=When%20someone%20with%20admin%20or%20owner%20permissions%20has%20enabled%20required%20reviews%2C%20they%20also%20can%20optionally%20require%20approval%20from%20a%20code%20owner%20before%20the%20author%20can%20merge%20a%20pull%20request%20in%20the%20repository.%20For%20more%20information%2C%20see%20%22About%20protected%20branches.%22

Reply:
Though apparently, it only works with write permission so I shall remove